.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+### NOTE: The paths in this file match `.gitignore` file patterns.
+### See:
+### - https://git-scm.com/docs/gitignore#_pattern_format
+### - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+
+# Global / default owners; a later match takes precedence (the last matching
+# pattern takes the most precedence).
+* @hardfinhq/go-codeowners
```

Comment:
Error here:

> Unknown owner on line 8: make sure the team @hardfinhq/go-codeowners exists, is publicly visible, and has write access to the repository

Reply:
`@hardfinhq/go-codeowners` is **NOT** a public team, so I'll make a public team (and give that team write access here)

Reply:
```suggestion
* @hardfinhq/public-go-codeowners
```